dashboard/app/models/lesson.rb

```diff
@@ -263,13 +263,23 @@ def localized_title
   end
 
   def localized_name
-    if script.lessons.many?
+    # The behavior to show the script title instead of the lesson name in
+    # single-lesson scripts is deprecated.
+    #
+    # TODO(dave): once all scripts are migrated and no longer
+    # using legacy lesson plans, remove this condition and consolidate with
```

Comment:
This wording is a little confusing. I think by "all scripts" you mean "all scripts with one lesson" is that right?

Reply:
I actually went back and forth on that :-) I could do this as soon as "all scripts with one lesson" have been migrated, or I could wait until "all scripts" have been migrated, which is when I was planning on doing the cleanup pass. I'd be happy to change it to "all scripts with one lesson" if that seems clearer.

Reply:
I was asking because I thought we didn't need to wait until everything was migrated to make this change. I thought this only impacted scripts with one lesson